packages/yoastseo/src/scoring/assessments/inclusiveLanguage/helpers/notInclusiveWhenStandalone.js

```diff
@@ -14,8 +19,8 @@ import { punctuationList } from "../../../../languageProcessing/helpers/sanitize
  */
 export default function notInclusiveWhenStandalone( words, nonInclusivePhrase ) {
```

Comment:
There are some spelling errors in the above JSDoc comment ("non inclusive" => "non-inclusive", "standalon" => "standalone"). Would be great to fix that right away.

Reply:
Also, the type of `nonNouns` is `string[]`, the current type allocation shows a warning in PhpStorm.

Reply:
Good points. That is so sloppy. 🙈